.storybook/main.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  stories: ['../src/**/*.stories.mdx', '../src/**/*.stories.@(js|jsx|ts|tsx)'],
```

Comment:
Let's remove those mdx files.
```suggestion
  stories: ['../src/**/*.stories.*'],
```